pkgs/top-level/all-packages.nix

```diff
@@ -33034,15 +33034,14 @@ with pkgs;
     inherit lua;
   };
 
-  shaka-packager = callPackage ../applications/video/shaka-packager { };
+  # Wrap avoiding rebuild
+  mpv = mpv-unwrapped.wrapper { mpv = mpv-unwrapped; };
 
-  # Wraps without triggering a rebuild
-  wrapMpv = callPackage ../applications/video/mpv/wrapper.nix { };
```

Comment:
Removal of wrapMpv breaks `pkgs/by-name/sv/svp/mvp.nix`

```
       error: evaluation aborted with the following error message: 'lib.customisation.callPackageWith: Function called without required argument "wrapMpv" at /home/scrumplex/.cache/nixpkgs-review/pr-304349/nixpkgs/pkgs/by-name/sv/svp/mpv.nix:3'
```

Reply:
Fixed.